src/pages/Admin/Venue/DetailsValidationSchema.ts

```diff
@@ -59,6 +59,7 @@ export const validationSchema = Yup.object()
     name: Yup.string()
       .required("Required")
```

Comment:
Unrelated to your change but would be great if these texts are replaced with more descriptive ones like in `src/pages/Admin/Details/ValidationSchema.ts`

Reply:
I can make a new PR for this

Reply:
See https://github.com/sparkletown/sparkle/pull/1297